.github/workflows/build.yml

```diff
@@ -0,0 +1,113 @@
+name: Build Tauri App
+
+on:
+  push:
+    tags:
+      - v*
+      - test*
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build-binaries:
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [self-hosted] # This is ubuntu, waiting on a windows runner
+    runs-on: ${{ matrix.platform }}
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      RUST_BACKTRACE: full
+      RUST_LOG: debug
+      ACTIONS_STEP_DEBUG: true
+    steps:
+      - name: Import Secrets
+        id: import-secrets
+        uses: hashicorp/vault-action@v2
+        with:
+          url: ${{ secrets.VAULT_URL }}
+          method: approle
+          roleId: ${{ secrets.VAULT_ROLE_ID }}
+          secretId: ${{ secrets.VAULT_SECRET_ID }}
+          secrets: |
+            kv/team/text/data/artifactory * | ART_;
+            kv/team/text/data/trokk-stage * | TROKK_
+
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies (ubuntu only)
+        if: matrix.platform == 'self-hosted'
```

Comment:
Ikke nøye for nå, men platform = self-hosted? Vil det ikke fort kunne være det samme for windows?

Reply:
Navnet her defineres av org-admin, dvs plattform. Vi får se hva de gjør når windows kommer opp.